src/public/app/widgets/buttons/global_menu.js

```diff
@@ -89,15 +108,35 @@ const TPL = `
             About Trilium Notes
         </a>
 
+        <a class="dropdown-item show-about-dialog-button">
+            <span class="bx bx-sync"></span>
+            Update to newest version
+        </a>
+
         <a class="dropdown-item logout-button" data-trigger-command="logout">
             <span class="bx bx-log-out"></span>
             Logout
         </a>
     </div>
 </div>
 `;
+const RELEASES_API_URL = "https://api.github.com/repos/zadam/trilium/releases/latest";
+const CURRENT_VERSION = process.env.npm_package_version;
 
 export default class GlobalMenuWidget extends BasicWidget {
+    static getVersionChange(oldVersion, newVersion) {
+        const [oldMajor, oldSemiMajor, oldMinor] = oldVersion.split(".").map(Number);
+        const [newMajor, newSemiMajor, newMinor] = newVersion.split(".").map(Number);
+
+        if (newMajor !== oldMajor) {
+            return "major";
+        } else if (newSemiMajor !== oldSemiMajor) {
+            return "semi-major";
+        } else if (newMinor !== oldMinor) {
+            return "minor";
+        }
+    }
```

Comment:
Standard terminology is MAJOR.MINOR.PATCH (see e.g. [this](https://semver.org/) even though Trilium does not follow semantic versioning strictly). But I don't think there's a need to differentiate between versions ...

Reply:
I would pledge to differentiate between versions because some users will probably not update a patch (because a patch often happens and most people find them annoying) but are welcome to minor are major versions (because they often contain new features)